sdk/azidentity/azidentity.go

```diff
@@ -54,18 +54,19 @@ func (e *AADAuthenticationFailedError) Error() string {
 	return msg
 }
 
-// AuthenticationFailedError is a struct used to marshal responses when authentication has failed
+// AuthenticationFailedError is returned when the authentication request has failed.
 type AuthenticationFailedError struct {
 	inner error
 	msg   string
 }
 
-// Unwrap method on AuthenticationFailedError provides access to the inner error
+// Unwrap method on AuthenticationFailedError provides access to the inner error authentication
```

Comment:
Will this always be an inner authentication error?  Maybe it's enough to say `provides access to the inner error if available.`

Reply:
Done